server.py

```diff
@@ -0,0 +1,55 @@
+import chat_proto.chat_pb2_grpc as chat_pb2_grpc
+import chat_proto.chat_pb2 as chat_pb2
+from concurrent import futures
+import grpc
+import time
+
+
+class ChatServiceServicer(chat_pb2_grpc.ChatServiceServicer):
+    
+
+    list_of_users = chat_pb2.Users()
+
+    list_of_messages = []
+
+
+    def sendMessage(self, request, context):
+
+        message = request.message
+        message.created_at = time.time()
+        ChatServiceServicer.list_of_messages.append(message)
+
+        # Check users if they exist in our system
+        if message.from_user not in ChatServiceServicer.list_of_users.user:
```

Comment:
It's better to use `set` of users instead of `list` of users. Our users should be unique anyway and `in` operation works faster for `set` than for `list`. You can google and read why it's so